### PR TITLE
Update CHANGES.md with release date

### PR DESCRIPTION
**Pull request by gberaudo**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Changelog
 
+## v1.3 - 2015-03-23
+
 * Port to Cesium 1.7.1 and Ol 3.3.0
 
 # Breaking changes
```
Comment (ahocevar) [inline CHANGES.md]: You'll want a different heading type for the breaking changes (e.g. `###`, or just a bullet point like for previous releases).

**Comment by ahocevar** [comment]
One comment as line note above. Can be merged when fixed.

**Comment by gberaudo** [comment]
You have a sharp eye @ahocevar, thank you for spotting this. I also moved up the breaking change section. I think we should always have it first, is it OK for you?

**Comment by ahocevar** [comment]
Fine with me. Looks good now.

**Pull request merged**